src/Stratis.Bitcoin.Features.Consensus/Rules/CommonRules/BlockSizeRule.cs

```diff
@@ -35,7 +35,7 @@ public override Task RunAsync(RuleContext context)
 
             // Size limits.
             if ((block.Transactions.Count == 0) || (block.Transactions.Count > options.MaxBlockBaseSize) || 
-                (GetSize(this.Parent.Network, block, context.Consensus.NetworkOptions & ~NetworkOptions.All) > options.MaxBlockBaseSize))
+                (GetSize(this.Parent.Network, block, TransactionOptions.None) > options.MaxBlockBaseSize))
```

Comment:
same as above

Reply:
Yeah I looked up what was there before and reverted back